app/components/daisy/data_display/figure_component.rb

```diff
@@ -43,12 +53,27 @@ def before_render
 
   def call
     part(:component) do
-      if @src
-        concat(part(:image))
+      if @position == "bottom"
+        # Show content first, then image
+        concat(content)
+        if @src
+          concat(part(:image))
+        end
```

Comment:
Should use the following:


```suggestion
        concat(part(:image)) if @src
```